src/components/Dailymotion.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import ReactResizeDetector from "react-resize-detector";
+import { fixHeightWithRatio, getRatio } from "../utils/Utils";
+
+export interface DailymotionProps {
+    url: string;
+    autoPlay: boolean;
+    controls: boolean;
+    muted: boolean;
+    aspectRatio?: boolean;
+}
+
+export class Dailymotion extends React.Component<DailymotionProps> {
```

Comment:
Missing typing for `state`
```suggestion
export class Dailymotion extends React.Component<DailymotionProps, DailymotionState> {
```